MultiVersions/Forge/main/meadow/lang/zh_tw.json

```
@@ -0,0 +1,249 @@
+  "block.meadow.mossy_limestone_brick_wall": "青苔石灰岩磚牆",
+  "block.meadow.mossy_limestone_bricks": "青苔石灰岩磚",
+  "block.meadow.oil_lantern": "油燈",
+  "block.meadow.patched_bed": "補丁羊毛床",
+  "block.meadow.patched_carpet": "補丁羊毛地毯",
+  "block.meadow.patched_wool": "補丁羊毛",
+  "block.meadow.pine_barn_door": "松木穀倉門",
+  "block.meadow.pine_barn_trapdoor": "松木穀倉地板門",
+  "block.meadow.pine_beam": "松木梁柱",
+  "block.meadow.pine_button": "松木按鈕",
+  "block.meadow.pine_door": "松木門",
+  "block.meadow.pine_fence": "松木柵欄",
+  "block.meadow.pine_fence_gate": "松木柵欄門",
+  "block.meadow.pine_hanging_sign": "懸掛式松木告示牌",
+  "block.meadow.pine_leaves": "松木樹葉",
+  "block.meadow.pine_leaves_2": "橙色松木樹葉",
+  "block.meadow.pine_log": "松木原木",
+  "block.meadow.pine_planks": "松木材",
+  "block.meadow.pine_pressure_plate": "松木壓力板",
+  "block.meadow.pine_railing": "松木圍欄",
+  "block.meadow.pine_sapling": "松木樹苗",
+  "block.meadow.pine_sign": "松木告示牌",
+  "block.meadow.pine_slab": "松木半磚",
+  "block.meadow.pine_stairs": "松木階梯",
+  "block.meadow.pine_trapdoor": "松木地板門",
+  "block.meadow.pine_window": "松木窗戶",
+  "block.meadow.pine_wood": "松木塊",
+  "block.meadow.polished_limestone_bricks": "拋光石灰岩",
+  "block.meadow.roasted_ham": "炙燒水牛火腿",
+  "block.meadow.rocky_bed": "岩石羊毛床",
+  "block.meadow.rocky_carpet": "岩石羊毛地毯",
+  "block.meadow.rocky_wool": "岩石羊毛",
+  "block.meadow.saxifrage": "虎耳草",
+  "block.meadow.sheep_cheese_block": "綿羊乳酪輪",
+  "block.meadow.shelf": "松木架",
+  "block.meadow.shutter_block": "窗戶隔板",
+  "block.meadow.shutter_block_berry": "彩繪窗戶隔板（莓果）",
+  "block.meadow.shutter_block_fir": "彩繪窗戶隔板（冷杉樹）",
+  "block.meadow.shutter_block_poppy": "彩繪窗戶隔板（罌粟）",
+  "block.meadow.small_fir": "小型冷杉樹",
+  "block.meadow.stone_bench": "石長椅",
+  "block.meadow.stone_table": "石桌",
+  "block.meadow.stove_tiles": "磚瓦爐灶",
+  "block.meadow.stove_tiles_bench": "磚瓦長凳",
+  "block.meadow.stove_tiles_lid": "磚瓦爐灶（擋板）",
+  "block.meadow.stove_tiles_wood": "磚瓦爐灶（柴火堆）",
+  "block.meadow.stove_tiles_wood_on": "磚瓦爐灶（燃燒中）",
+  "block.meadow.straw_bed": "稻草床",
+  "block.meadow.stripped_pine_log": "剝皮松木原木",
+  "block.meadow.stripped_pine_wood": "剝皮松木塊",
+  "block.meadow.sun_patterned_window": "太陽花紋窗戶",
+  "block.meadow.table": "松木桌",
+  "block.meadow.umbra_bed": "陰影牛毛床",
+  "block.meadow.umbra_carpet": "陰影牛毛地毯",
+  "block.meadow.umbra_wool": "陰影牛毛",
+  "block.meadow.warped_bed": "扭曲牛毛床",
+  "block.meadow.warped_carpet": "扭曲牛毛地毯",
+  "block.meadow.warped_cheese_block": "扭曲乳酪輪",
+  "block.meadow.warped_wool": "扭曲牛毛",
+  "block.meadow.watering_can": "澆水壺",
+  "block.meadow.wheelbarrow": "手推車",
+  "block.meadow.woodcutter": "伐木樁",
+  "block.meadow.wooden_cauldron": "木製鍋釜",
+  "block.meadow.wooden_flower_box": "木製花箱",
+  "block.meadow.wooden_flower_pot_big": "大型木製花盆",
+  "block.meadow.wooden_flower_pot_small": "木製花盆",
+  "block.meadow.wooden_powder_snow_cauldron": "裝粉雪的木製鍋釜",
+  "block.meadow.wooden_water_cauldron": "裝水的木製鍋釜",
+  "creativetab.meadow.tab": "[Let's Do]Meadow",
```

模組名稱沿用 MCMOD，如果不想翻譯物品群組，可以略過
```suggestion
  "creativetab.meadow.tab": "共度時光｜青青草甸",
```